tools/publish-packages.js

```diff
@@ -639,6 +639,22 @@ async function _updateAndroidDependenciesAsync(allConfigs) {
     }
     console.log();
   }
+
+  if (await _promptAsync(`Do you want to update dependencies in ${chalk.magenta('expo/android/app/build.gradle')}?`)) {
```

Comment:
well, it's actually the same as updating `android/expoview/build.gradle` so maybe we could make a separate function for this 🤔